Tests failing due to unexpected behaviour of repr()

**Description of the problem**

Tests are failing on `master` when checking that `BaseGridder` representation is obtained as expected through the `repr()` built-in function.
Seems like `repr()` is not adding the default arguments to the string.

**Full code that generated the error**

<!--
Include any data files or inputs required to run the code. It really helps if
we can run the code on our own machines.
-->

```bash
make test
```


**Full error message**

```
# Run a tmp folder to make sure the tests are run on the installed version
mkdir -p tmp-test-dir-with-unique-name
cd tmp-test-dir-with-unique-name; MPLBACKEND='agg' pytest --cov-config=../.coveragerc --cov-report=term-missing --cov=verde --doctest-modules -v --pyargs verde
======================================================= test session starts ========================================================
platform linux -- Python 3.7.6, pytest-5.4.2, py-1.8.0, pluggy-0.13.0 -- /home/santi/.miniconda3/envs/verde/bin/python
cachedir: .pytest_cache
Matplotlib: 3.1.2
Freetype: 2.10.0
rootdir: /home/santi/git/verde
plugins: cov-2.9.0, mpl-0.11
collected 152 items                                                                                                                

../verde/coordinates.py::verde.coordinates.block_split PASSED                                                                [  0%]
../verde/coordinates.py::verde.coordinates.expanding_window PASSED                                                           [  1%]
../verde/coordinates.py::verde.coordinates.get_region PASSED                                                                 [  1%]
../verde/coordinates.py::verde.coordinates.grid_coordinates PASSED                                                           [  2%]
../verde/coordinates.py::verde.coordinates.inside PASSED                                                                     [  3%]
../verde/coordinates.py::verde.coordinates.longitude_continuity PASSED                                                       [  3%]
../verde/coordinates.py::verde.coordinates.pad_region PASSED                                                                 [  4%]
../verde/coordinates.py::verde.coordinates.profile_coordinates PASSED                                                        [  5%]
../verde/coordinates.py::verde.coordinates.rolling_window PASSED                                                             [  5%]
../verde/coordinates.py::verde.coordinates.scatter_points PASSED                                                             [  6%]
../verde/coordinates.py::verde.coordinates.shape_to_spacing PASSED                                                           [  7%]
../verde/distances.py::verde.distances.median_distance PASSED                                                                [  7%]
../verde/mask.py::verde.mask.convexhull_mask PASSED                                                                          [  8%]
../verde/mask.py::verde.mask.distance_mask PASSED                                                                            [  9%]
../verde/model_selection.py::verde.model_selection.BlockShuffleSplit PASSED                                                  [  9%]
../verde/model_selection.py::verde.model_selection.cross_val_score PASSED                                                    [ 10%]
../verde/model_selection.py::verde.model_selection.select PASSED                                                             [ 11%]
../verde/model_selection.py::verde.model_selection.train_test_split PASSED                                                   [ 11%]
../verde/projections.py::verde.projections.project_region PASSED                                                             [ 12%]
../verde/trend.py::verde.trend.Trend PASSED                                                                                  [ 13%]
../verde/trend.py::verde.trend.Trend.jacobian PASSED                                                                         [ 13%]
../verde/trend.py::verde.trend.polynomial_power_combinations PASSED                                                          [ 14%]
../verde/utils.py::verde.utils.grid_to_table PASSED                                                                          [ 15%]
../verde/utils.py::verde.utils.maxabs PASSED                                                                                 [ 15%]
../verde/utils.py::verde.utils.variance_to_weights PASSED                                                                    [ 16%]
../verde/base/base_classes.py::verde.base.base_classes.BaseGridder FAILED                                                    [ 17%]
../verde/base/base_classes.py::verde.base.base_classes.get_data_names PASSED                                                 [ 17%]
../verde/base/utils.py::verde.base.utils.check_data PASSED                                                                   [ 18%]
../verde/base/utils.py::verde.base.utils.n_1d_arrays PASSED                                                                  [ 19%]
../verde/datasets/synthetic.py::verde.datasets.synthetic.CheckerBoard PASSED                                                 [ 19%]
../verde/tests/test_base.py::test_check_coordinates PASSED                                                                   [ 20%]
../verde/tests/test_base.py::test_get_dims PASSED                                                                            [ 21%]
../verde/tests/test_base.py::test_get_data_names PASSED                                                                      [ 21%]
../verde/tests/test_base.py::test_get_data_names_fails PASSED                                                                [ 22%]
../verde/tests/test_base.py::test_get_instance_region PASSED                                                                 [ 23%]
../verde/tests/test_base.py::test_basegridder FAILED                                                                         [ 23%]
../verde/tests/test_base.py::test_basegridder_projection PASSED                                                              [ 24%]
../verde/tests/test_base.py::test_check_fit_input PASSED                                                                     [ 25%]
../verde/tests/test_base.py::test_check_fit_input_fails_coordinates PASSED                                                   [ 25%]
../verde/tests/test_base.py::test_check_fit_input_fails_weights PASSED                                                       [ 26%]
../verde/tests/test_base.py::test_least_squares_copy_jacobian PASSED                                                         [ 26%]
../verde/tests/test_blockreduce.py::test_block_reduce PASSED                                                                 [ 27%]
../verde/tests/test_blockreduce.py::test_block_reduce_shape PASSED                                                           [ 28%]
../verde/tests/test_blockreduce.py::test_block_reduce_scatter PASSED                                                         [ 28%]
../verde/tests/test_blockreduce.py::test_block_reduce_weights PASSED                                                         [ 29%]
../verde/tests/test_blockreduce.py::test_block_reduce_drop_coords PASSED                                                     [ 30%]
../verde/tests/test_blockreduce.py::test_block_reduce_multiple_coordinates PASSED                                            [ 30%]
../verde/tests/test_blockreduce.py::test_block_reduce_scatter_multiple_coordinates PASSED                                    [ 31%]
../verde/tests/test_blockreduce.py::test_block_reduce_multiple_components PASSED                                             [ 32%]
../verde/tests/test_blockreduce.py::test_block_reduce_multiple_weights PASSED                                                [ 32%]
../verde/tests/test_blockreduce.py::test_blockmean_noweights PASSED                                                          [ 33%]
../verde/tests/test_blockreduce.py::test_blockmean_noweights_multiple_components PASSED                                      [ 34%]
../verde/tests/test_blockreduce.py::test_blockmean_noweights_table PASSED                                                    [ 34%]
../verde/tests/test_blockreduce.py::test_blockmean_uncertainty_weights PASSED                                                [ 35%]
../verde/tests/test_blockreduce.py::test_blockmean_variance_weights PASSED                                                   [ 36%]
../verde/tests/test_chain.py::test_chain PASSED                                                                              [ 36%]
../verde/tests/test_chain.py::test_chain_double PASSED                                                                       [ 37%]
../verde/tests/test_chain.py::test_chain_vector PASSED                                                                       [ 38%]
../verde/tests/test_coordinates.py::test_rolling_window_invalid_coordinate_shapes PASSED                                     [ 38%]
../verde/tests/test_coordinates.py::test_rolling_window_empty PASSED                                                         [ 39%]
../verde/tests/test_coordinates.py::test_rolling_window_warnings PASSED                                                      [ 40%]
../verde/tests/test_coordinates.py::test_spacing_to_shape PASSED                                                             [ 40%]
../verde/tests/test_coordinates.py::test_spacing_to_shape_fails PASSED                                                       [ 41%]
../verde/tests/test_coordinates.py::test_grid_coordinates_fails PASSED                                                       [ 42%]
../verde/tests/test_coordinates.py::test_check_region PASSED                                                                 [ 42%]
../verde/tests/test_coordinates.py::test_profile_coordiantes_fails PASSED                                                    [ 43%]
../verde/tests/test_coordinates.py::test_longitude_continuity PASSED                                                         [ 44%]
../verde/tests/test_coordinates.py::test_invalid_geographic_region PASSED                                                    [ 44%]
../verde/tests/test_coordinates.py::test_invalid_geographic_coordinates PASSED                                               [ 45%]
../verde/tests/test_datasets.py::test_datasets_locate PASSED                                                                 [ 46%]
../verde/tests/test_datasets.py::test_fetch_texas_wind PASSED                                                                [ 46%]
../verde/tests/test_datasets.py::test_setup_texas_wind PASSED                                                                [ 47%]
../verde/tests/test_datasets.py::test_fetch_baja_bathymetry PASSED                                                           [ 48%]
../verde/tests/test_datasets.py::test_setup_baja_bathymetry PASSED                                                           [ 48%]
../verde/tests/test_datasets.py::test_fetch_rio_magnetic PASSED                                                              [ 49%]
../verde/tests/test_datasets.py::test_setup_rio_magnetic PASSED                                                              [ 50%]
../verde/tests/test_datasets.py::test_fetch_california_gps PASSED                                                            [ 50%]
../verde/tests/test_datasets.py::test_setup_california_gps PASSED                                                            [ 51%]
../verde/tests/test_distances.py::test_distance_nearest PASSED                                                               [ 51%]
../verde/tests/test_distances.py::test_distance_k_nearest PASSED                                                             [ 52%]
../verde/tests/test_distances.py::test_distance_nearest_projection PASSED                                                    [ 53%]
../verde/tests/test_io.py::test_load_surfer_header PASSED                                                                    [ 53%]
../verde/tests/test_io.py::test_load_surfer PASSED                                                                           [ 54%]
../verde/tests/test_io.py::test_load_surfer_nans PASSED                                                                      [ 55%]
../verde/tests/test_io.py::test_load_surfer_file PASSED                                                                      [ 55%]
../verde/tests/test_io.py::test_load_surfer_integrity PASSED                                                                 [ 56%]
../verde/tests/test_mask.py::test_convexhull_mask PASSED                                                                     [ 57%]
../verde/tests/test_mask.py::test_convexhull_mask_projection PASSED                                                          [ 57%]
../verde/tests/test_mask.py::test_distance_mask PASSED                                                                       [ 58%]
../verde/tests/test_mask.py::test_distance_mask_projection PASSED                                                            [ 59%]
../verde/tests/test_mask.py::test_distance_mask_grid PASSED                                                                  [ 59%]
../verde/tests/test_mask.py::test_distance_mask_missing_args PASSED                                                          [ 60%]
../verde/tests/test_mask.py::test_distance_mask_wrong_shapes PASSED                                                          [ 61%]
../verde/tests/test_minimal.py::test_minimal_integration_2d_gps PASSED                                                       [ 61%]
../verde/tests/test_model_selection.py::test_cross_val_score_client PASSED                                                   [ 62%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_n_splits PASSED                                               [ 63%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_fails_balancing PASSED                                        [ 63%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_fails_spacing_shape PASSED                                    [ 64%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_fails_data_shape PASSED                                       [ 65%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.1] PASSED                                         [ 65%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.2] PASSED                                         [ 66%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.3] PASSED                                         [ 67%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.4] PASSED                                         [ 67%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.5] PASSED                                         [ 68%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.6] PASSED                                         [ 69%]
../verde/tests/test_model_selection.py::test_blockshufflesplit_balancing[0.9] PASSED                                         [ 69%]
../verde/tests/test_projections.py::test_project_grid[nearest] PASSED                                                        [ 70%]
../verde/tests/test_projections.py::test_project_grid[linear] PASSED                                                         [ 71%]
../verde/tests/test_projections.py::test_project_grid[cubic] PASSED                                                          [ 71%]
../verde/tests/test_projections.py::test_project_grid[gridder] PASSED                                                        [ 72%]
../verde/tests/test_projections.py::test_project_grid_name PASSED                                                            [ 73%]
../verde/tests/test_projections.py::test_project_grid_antialias[True] PASSED                                                 [ 73%]
../verde/tests/test_projections.py::test_project_grid_antialias[False] PASSED                                                [ 74%]
../verde/tests/test_projections.py::test_project_grid_fails_dataset PASSED                                                   [ 75%]
../verde/tests/test_projections.py::test_project_grid_fails_dimensions[1] PASSED                                             [ 75%]
../verde/tests/test_projections.py::test_project_grid_fails_dimensions[3] PASSED                                             [ 76%]
../verde/tests/test_scipy.py::test_scipy_gridder_same_points PASSED                                                          [ 76%]
../verde/tests/test_scipy.py::test_scipy_gridder PASSED                                                                      [ 77%]
../verde/tests/test_scipy.py::test_scipy_gridder_region PASSED                                                               [ 78%]
../verde/tests/test_scipy.py::test_scipy_gridder_extra_args PASSED                                                           [ 78%]
../verde/tests/test_scipy.py::test_scipy_gridder_fails PASSED                                                                [ 79%]
../verde/tests/test_scipy.py::test_scipy_gridder_warns PASSED                                                                [ 80%]
../verde/tests/test_spline.py::test_spline_cv[serial] PASSED                                                                 [ 80%]
../verde/tests/test_spline.py::test_spline_cv[delayed] PASSED                                                                [ 81%]
../verde/tests/test_spline.py::test_spline_cv[distributed] PASSED                                                            [ 82%]
../verde/tests/test_spline.py::test_spline PASSED                                                                            [ 82%]
../verde/tests/test_spline.py::test_spline_weights PASSED                                                                    [ 83%]
../verde/tests/test_spline.py::test_spline_region PASSED                                                                     [ 84%]
../verde/tests/test_spline.py::test_spline_damping PASSED                                                                    [ 84%]
../verde/tests/test_spline.py::test_spline_warns_weights PASSED                                                              [ 85%]
../verde/tests/test_spline.py::test_spline_warns_underdetermined PASSED                                                      [ 86%]
../verde/tests/test_spline.py::test_spline_jacobian_implementations PASSED                                                   [ 86%]
../verde/tests/test_spline.py::test_spline_predict_implementations PASSED                                                    [ 87%]
../verde/tests/test_trend.py::test_trend PASSED                                                                              [ 88%]
../verde/tests/test_trend.py::test_trend_mean PASSED                                                                         [ 88%]
../verde/tests/test_trend.py::test_trend_weights PASSED                                                                      [ 89%]
../verde/tests/test_trend.py::test_polynomial_combinations_fails PASSED                                                      [ 90%]
../verde/tests/test_trend.py::test_trend_jacobian_fails PASSED                                                               [ 90%]
../verde/tests/test_utils.py::test_parse_engine PASSED                                                                       [ 91%]
../verde/tests/test_utils.py::test_parse_engine_fails PASSED                                                                 [ 92%]
../verde/tests/test_utils.py::test_dummy_jit PASSED                                                                          [ 92%]
../verde/tests/test_utils.py::test_kdtree PASSED                                                                             [ 93%]
../verde/tests/test_utils.py::test_grid_to_table_order PASSED                                                                [ 94%]
../verde/tests/test_vector.py::test_vector2d PASSED                                                                          [ 94%]
../verde/tests/test_vector.py::test_vector2d_weights PASSED                                                                  [ 95%]
../verde/tests/test_vector.py::test_vector2d_fails PASSED                                                                    [ 96%]
../verde/tests/test_vector.py::test_vector2d_jacobian_implementations PASSED                                                 [ 96%]
../verde/tests/test_vector.py::test_vector2d_predict_implementations PASSED                                                  [ 97%]
../verde/tests/test_vector.py::test_vector_trend PASSED                                                                      [ 98%]
../verde/tests/test_vector.py::test_vector_trend_3d PASSED                                                                   [ 98%]
../verde/tests/test_vector.py::test_vector_trend_fails PASSED                                                                [ 99%]
../verde/tests/test_vector.py::test_vector_trend_weights PASSED                                                              [100%]

============================================================= FAILURES =============================================================
__________________________________________ [doctest] verde.base.base_classes.BaseGridder ___________________________________________
062     ...         check_is_fitted(self, ['mean_'])
063     ...         return np.ones_like(coordinates[0])*self.mean_
064     >>> # Try it on some synthetic data
065     >>> synthetic = vd.datasets.CheckerBoard(region=(0, 5, -10, 8))
066     >>> data = synthetic.scatter()
067     >>> print('{:.4f}'.format(data.scalars.mean()))
068     -32.2182
069     >>> # Fit the gridder to our synthetic data
070     >>> grd = MeanGridder().fit((data.easting, data.northing), data.scalars)
071     >>> grd
Expected:
    MeanGridder(multiplier=1)
Got:
    MeanGridder()

/home/santi/git/verde/verde/base/base_classes.py:71: DocTestFailure
_________________________________________________________ test_basegridder _________________________________________________________

    def test_basegridder():
        "Test basic functionality of BaseGridder"
    
        with pytest.raises(NotImplementedError):
            BaseGridder().predict(None)
        with pytest.raises(NotImplementedError):
            BaseGridder().fit(None, None)
    
        grd = PolyGridder()
>       assert repr(grd) == "PolyGridder(degree=1)"
E       AssertionError: assert 'PolyGridder()' == 'PolyGridder(degree=1)'
E         - PolyGridder(degree=1)
E         ?             --------
E         + PolyGridder()

../verde/tests/test_base.py:114: AssertionError
========================================================= warnings summary =========================================================
/home/santi/.miniconda3/envs/verde/lib/python3.7/site-packages/numba/types/containers.py:3
  /home/santi/.miniconda3/envs/verde/lib/python3.7/site-packages/numba/types/containers.py:3: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Iterable

verde/tests/test_datasets.py::test_fetch_rio_magnetic
  /home/santi/git/verde/verde/datasets/sample_data.py:184: DeprecationWarning: The Rio magnetic anomaly dataset is deprecated and will be removed in Verde v2.0.0. Use a different dataset instead.
    DeprecationWarning,

verde/tests/test_datasets.py::test_setup_rio_magnetic
  /home/santi/git/verde/verde/datasets/sample_data.py:221: DeprecationWarning: The Rio magnetic anomaly dataset is deprecated and will be removed in Verde v2.0.0. Use a different dataset instead.
    DeprecationWarning,

verde/tests/test_model_selection.py::test_cross_val_score_client
  /home/santi/.miniconda3/envs/verde/lib/python3.7/site-packages/distributed/dashboard/core.py:72: UserWarning: 
  Port 8787 is already in use. 
  Perhaps you already have a cluster running?
  Hosting the diagnostics dashboard on a random port instead.
    warnings.warn("\n" + msg)

verde/tests/test_model_selection.py::test_cross_val_score_client
  /home/santi/git/verde/verde/model_selection.py:563: DeprecationWarning: The 'client' parameter of 'verde.cross_val_score' is deprecated and will be removed in Verde 2.0.0. Use the 'delayed' parameter instead.
    DeprecationWarning,

verde/tests/test_spline.py::test_spline_cv[distributed]
  /home/santi/git/verde/verde/spline.py:148: DeprecationWarning: The 'client' parameter of 'verde.SplineCV' is deprecated and will be removed in Verde 2.0.0. Use the 'delayed' parameter instead.
    DeprecationWarning,

verde/tests/test_vector.py::test_vector2d
verde/tests/test_vector.py::test_vector2d_weights
verde/tests/test_vector.py::test_vector2d_fails
verde/tests/test_vector.py::test_vector2d_jacobian_implementations
verde/tests/test_vector.py::test_vector2d_jacobian_implementations
verde/tests/test_vector.py::test_vector2d_predict_implementations
verde/tests/test_vector.py::test_vector2d_predict_implementations
  /home/santi/git/verde/verde/vector.py:237: DeprecationWarning: VectorSpline2D is deprecated and will be removed in Verde v2.0.0. Please use the implementation in the Erizo package instead (https://github.com/fatiando/erizo).
    DeprecationWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html

----------- coverage: platform linux, python 3.7.6-final-0 -----------
Name                                                  Stmts   Miss Branch BrPart  Cover   Missing
-------------------------------------------------------------------------------------------------
/home/santi/git/verde/verde/base/__init__.py              3      0      0      0   100%
/home/santi/git/verde/verde/base/base_classes.py         81      0     32      0   100%
/home/santi/git/verde/verde/base/least_squares.py        14      0      4      0   100%
/home/santi/git/verde/verde/base/utils.py                31      0     30      0   100%
/home/santi/git/verde/verde/blockreduce.py               99      0     63      0   100%
/home/santi/git/verde/verde/chain.py                     29      0     14      0   100%
/home/santi/git/verde/verde/coordinates.py              175      0     88      0   100%
/home/santi/git/verde/verde/datasets/__init__.py          2      0      0      0   100%
/home/santi/git/verde/verde/datasets/sample_data.py      58      2      8      0    97%   15-16
/home/santi/git/verde/verde/datasets/synthetic.py        26      0      4      0   100%
/home/santi/git/verde/verde/distances.py                 12      0      2      0   100%
/home/santi/git/verde/verde/io.py                        37      0     20      0   100%
/home/santi/git/verde/verde/mask.py                      42      0     22      0   100%
/home/santi/git/verde/verde/model_selection.py           75      0     33      0   100%
/home/santi/git/verde/verde/projections.py               38      0     12      0   100%
/home/santi/git/verde/verde/scipygridder.py              30      0      6      0   100%
/home/santi/git/verde/verde/spline.py                   125     13     34      0    87%   18-20, 541-548, 554-559
/home/santi/git/verde/verde/trend.py                     40      0     10      0   100%
/home/santi/git/verde/verde/utils.py                     76      4     30      0    96%   13-14, 18-19
/home/santi/git/verde/verde/vector.py                   123     22     37      1    81%   17-19, 286->288, 449-459, 465-476
/home/santi/git/verde/verde/version.py                    4      0      0      0   100%
-------------------------------------------------------------------------------------------------
TOTAL                                                  1120     41    449      1    96%

===================================================== short test summary info ======================================================
FAILED ../verde/base/base_classes.py::verde.base.base_classes.BaseGridder
FAILED ../verde/tests/test_base.py::test_basegridder - AssertionError: assert 'PolyGridder()' == 'PolyGridder(degree=1)'
=========================================== 2 failed, 150 passed, 13 warnings in 49.66s ============================================
make: *** [Makefile:26: test] Error 1

```



**System information**

* Operating system: Manjaro
* Python installation (Anaconda, system, ETS): Anaconda
* Version of Python: 3.7.6
* Version of this package: v1.4.0+7.g644037e
* If using conda, paste the output of `conda list` below:

<details>
<summary>output of conda list</summary>
<pre>

_libgcc_mutex             0.1                        main    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
appdirs                   1.4.3                      py_1    conda-forge
astroid                   2.3.3                    py37_0    conda-forge
attrs                     19.3.0                     py_0    conda-forge
babel                     2.7.0                      py_0    conda-forge
black                     19.10b0                  py37_0    conda-forge
bokeh                     1.4.0                    py37_0    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
ca-certificates           2020.4.5.1           hecc5488_0    conda-forge
cartopy                   0.17.0          py37h423102d_1009    conda-forge
certifi                   2020.4.5.1       py37hc8dfbb8_0    conda-forge
cffi                      1.13.2           py37h8022711_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.0                        py_0    conda-forge
cloudpickle               1.2.2                      py_1    conda-forge
coverage                  5.1              py37h8f50634_0    conda-forge
cryptography              2.8              py37h72c5cf5_1    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cytoolz                   0.10.1           py37h516909a_0    conda-forge
dask                      2.9.0                      py_0    conda-forge
dask-core                 2.9.0                      py_0    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
distributed               2.9.0                      py_0    conda-forge
docutils                  0.15.2                   py37_0    conda-forge
entrypoints               0.3                   py37_1000    conda-forge
expat                     2.2.5             he1b5a44_1004    conda-forge
flake8                    3.7.9            py37hc8dfbb8_1    conda-forge
fontconfig                2.13.1            h86ecdb6_1001    conda-forge
freetype                  2.10.0               he983fc9_1    conda-forge
fsspec                    0.6.1                      py_0    conda-forge
geos                      3.8.0                he1b5a44_0    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
glib                      2.58.3          py37h6f030ca_1002    conda-forge
gst-plugins-base          1.14.5               h0935bb2_0    conda-forge
gstreamer                 1.14.5               h36ae1b5_0    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
icu                       64.2                 he1b5a44_1    conda-forge
idna                      2.8                   py37_1000    conda-forge
imagesize                 1.1.0                      py_0    conda-forge
importlib_metadata        1.2.0                    py37_0    conda-forge
isort                     4.3.21                   py37_0    conda-forge
jinja2                    2.10.3                     py_0    conda-forge
joblib                    0.14.0                     py_0    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
kiwisolver                1.1.0            py37hc9558a2_0    conda-forge
lazy-object-proxy         1.4.3            py37h516909a_0    conda-forge
ld_impl_linux-64          2.33.1               h53a641e_7    conda-forge
libblas                   3.8.0               14_openblas    conda-forge
libcblas                  3.8.0               14_openblas    conda-forge
libclang                  9.0.0           default_hde54327_4    conda-forge
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 9.2.0                hdf63c60_0    conda-forge
libgfortran-ng            7.3.0                hdf63c60_2    conda-forge
libiconv                  1.15              h516909a_1005    conda-forge
liblapack                 3.8.0               14_openblas    conda-forge
libllvm8                  8.0.1                hc9558a2_0    conda-forge
libllvm9                  9.0.0                hc9558a2_3    conda-forge
libopenblas               0.3.7                h5ec1e0e_5    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_0    conda-forge
libtiff                   4.1.0                hc3755c2_1    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxkbcommon              0.9.1                hebb1f50_0    conda-forge
libxml2                   2.9.10               hee79883_0    conda-forge
llvmlite                  0.30.0           py37h8b12597_1    conda-forge
locket                    0.2.0                      py_2    conda-forge
lz4-c                     1.8.3             he1b5a44_1001    conda-forge
markupsafe                1.1.1            py37h516909a_0    conda-forge
matplotlib                3.1.2                    py37_1    conda-forge
matplotlib-base           3.1.2            py37h250f245_1    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
more-itertools            8.0.2                      py_0    conda-forge
msgpack-python            0.6.2            py37hc9558a2_0    conda-forge
mypy_extensions           0.4.3                    py37_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
nose                      1.3.7                 py37_1003    conda-forge
nspr                      4.24                 he1b5a44_0    conda-forge
nss                       3.47                 he751ad9_0    conda-forge
numba                     0.46.0           py37hb3f55d8_1    conda-forge
numpy                     1.18.4           py37h8960a57_0    conda-forge
numpydoc                  1.0.0              pyh9f0ad1d_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openssl                   1.1.1g               h516909a_0    conda-forge
owslib                    0.19.0                     py_1    conda-forge
packaging                 19.2                       py_0    conda-forge
pandas                    1.0.4            py37h0da4684_0    conda-forge
partd                     1.0.0                      py_0    conda-forge
pathspec                  0.6.0                      py_0    conda-forge
pcre                      8.43                 he1b5a44_0    conda-forge
pillow                    6.2.1            py37hd70f55b_1    conda-forge
pip                       20.1.1                     py_1    conda-forge
pluggy                    0.13.0                   py37_0    conda-forge
pooch                     1.1.1                      py_0    conda-forge
proj                      6.2.1                hc80f0dc_0    conda-forge
psutil                    5.6.7            py37h516909a_0    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
py                        1.8.0                      py_0    conda-forge
pycodestyle               2.5.0                      py_0    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyepsg                    0.4.0                      py_0    conda-forge
pyflakes                  2.1.1                      py_0    conda-forge
pygments                  2.5.2                      py_0    conda-forge
pykdtree                  1.3.1           py37h03ebfcd_1003    conda-forge
pylint                    2.4.4                    py37_0    conda-forge
pyopenssl                 19.1.0                   py37_0    conda-forge
pyparsing                 2.4.5                      py_0    conda-forge
pyproj                    2.4.2.post1      py37h12732c1_0    conda-forge
pyqt                      5.12.3           py37hcca6a23_1    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pyshp                     2.1.0                      py_0    conda-forge
pysocks                   1.7.1                    py37_0    conda-forge
pytest                    5.4.2            py37hc8dfbb8_0    conda-forge
pytest-cov                2.9.0              pyh9f0ad1d_0    conda-forge
pytest-mpl                0.11                       py_1    conda-forge
python                    3.7.6           h8356626_5_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytz                      2019.3                     py_0    conda-forge
pyyaml                    5.2              py37h516909a_0    conda-forge
qt                        5.12.5               hd8c4c69_1    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
regex                     2019.12.9        py37h516909a_0    conda-forge
requests                  2.22.0                   py37_1    conda-forge
scikit-learn              0.23.1           py37h8a51577_0    conda-forge
scipy                     1.4.1            py37ha3d9a3c_3    conda-forge
setuptools                42.0.2                   py37_0    conda-forge
shapely                   1.6.4           py37h5d51c17_1007    conda-forge
six                       1.13.0                   py37_0    conda-forge
snowballstemmer           2.0.0                      py_0    conda-forge
sortedcontainers          2.1.0                      py_0    conda-forge
sphinx                    2.2.1                      py_0    conda-forge
sphinx-gallery            0.7.0                      py_0    conda-forge
sphinx_rtd_theme          0.4.3                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.1                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.1                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.2                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.2                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.3                      py_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
tblib                     1.4.0                      py_0    conda-forge
threadpoolctl             2.0.0              pyh5ca1d4c_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
toml                      0.10.0                     py_0    conda-forge
toolz                     0.10.0                     py_0    conda-forge
tornado                   6.0.3            py37h516909a_0    conda-forge
typed-ast                 1.4.0            py37h516909a_0    conda-forge
typing_extensions         3.7.4.1                  py37_0    conda-forge
urllib3                   1.25.7                   py37_0    conda-forge
verde                     1.4.0+7.g644037e           dev_0    <develop>
wcwidth                   0.1.7                      py_1    conda-forge
wheel                     0.33.6                   py37_0    conda-forge
wrapt                     1.11.2           py37h516909a_0    conda-forge
xarray                    0.15.1                     py_0    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
yaml                      0.2.2                h516909a_1    conda-forge
zict                      1.0.0                      py_0    conda-forge
zipp                      0.6.0                      py_0    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
zstd                      1.4.4                h3b9ef0a_1    conda-forge


</pre>
</details>

--- a/verde/base/base_classes.py
+++ b/verde/base/base_classes.py
@@ -69,7 +69,7 @@ class BaseGridder(BaseEstimator):
     >>> # Fit the gridder to our synthetic data
     >>> grd = MeanGridder().fit((data.easting, data.northing), data.scalars)
     >>> grd
-    MeanGridder(multiplier=1)
+    MeanGridder()
     >>> # Interpolate on a regular grid
     >>> grid = grd.grid(region=(0, 5, -10, -8), shape=(30, 20))
     >>> np.allclose(grid.scalars, -32.2182)

--- a/verde/tests/test_base.py
+++ b/verde/tests/test_base.py
@@ -111,7 +111,7 @@ def test_basegridder():
         BaseGridder().fit(None, None)
 
     grd = PolyGridder()
-    assert repr(grd) == "PolyGridder(degree=1)"
+    assert repr(grd) == "PolyGridder()"
     grd.degree = 2
     assert repr(grd) == "PolyGridder(degree=2)"
 

